ml4h/arguments.py

```diff
@@ -269,6 +272,11 @@ def parse_args():
         help='If true saves the model weights from the last training epoch, otherwise the model with best validation loss is saved.',
     )
 
+    # 2D image data augmentation parameters
+    parser.add_argument('--rotation_factor', default=0., type=float, help='a float represented as fraction of 2 Pi, e.g., rotation_factor = 0.014 results in an output rotated by a random amount in the range [-5 degrees, 5 degrees]')
```

Comment:
Add that these are data augmentation specific to the help strings, eg "For data augmentation, a float....'